### PR TITLE
MC-15373: add approve invoice doc

### PR DESCRIPTION
**Pull request by tylerrwatsonn**
#### Changes made
added docs to approve invoice

#### Related PRs
https://github.com/cloudops/cloudmc-core/pull/1418

### CODE REVIEW

File: source/includes/administration/_invoices.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Attributes | &nbsp;
 Retrieve the list of invoices associated with an organization.
 
 ```shell
-# Retrieve applied pricing list
+# Retrieve list of invoices of an organization
```
Comment (dividedbyzero360) [inline source/includes/administration/_invoices.md:105]: Nice catch!

File: source/includes/administration/_invoices.md [hunk 2]
```diff
@@ -184,3 +184,90 @@ Attributes | &nbsp;
 `detail.categories.products.unit`<br/>*Object* | The unit object of the product.
 `detail.categories.products.unit.unit`<br/>*Object* | The unit value of the product.
 `detail.categories.products.unit.name`<br/>*Object* | The name of the unit of the product in the required language. Only present when defining custom units.
+
+<!-------------------- APPROVE INVOICE -------------------->
+### Approve invoice
+
+`PUT /invoices/:invoice_id/approve`
+
+Manually approve a draft invoice and issue the invoice to the customer by email.
```
Comment (dividedbyzero360) [inline source/includes/administration/_invoices.md]: Do you think we should add what will happen if the invoice is not in `draft` state?

File: source/includes/administration/_invoices.md [hunk 3]
```diff
@@ -184,3 +184,90 @@ Attributes | &nbsp;
...
+		},
+		"total": 272154.29,
+	}
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the invoice.
+`status`<br/>*enum* | The status of the invoice. Possible values are: USAGE_PENDING, DRAFT, ISSUED, OVERDUE, PAID, VOID.
```
Comment (dividedbyzero360) [inline source/includes/administration/_invoices.md]: Possible value is `ISSUED`?

Comment (tylerrwatsonn) [reply source/includes/administration/_invoices.md]: good catch!

File: source/includes/administration/_invoices.md [hunk 4]
```diff
@@ -184,3 +184,90 @@ Attributes | &nbsp;
...
+`createdDate`<br/>*date* | The created date of the invoice.
+`draftedDate`<br/>*date* | The date the invoice was drafted. Is null if status is still USAGE_PENDING.
```
Comment (dividedbyzero360) [inline source/includes/administration/_invoices.md]: We should remove _Is null if status is still USAGE_PENDING._ WDYT?

**Review by dividedbyzero360 — APPROVED** [review]
Minor comments. LGTM!

File: source/includes/administration/_invoices.md (hunk 2 above)
Comment (tylerrwatsonn) [reply source/includes/administration/_invoices.md]: https://github.com/cloudops/cloudmc-api-docs/pull/418/commits/f116010e0dd55c35d39fda4a440e0d90944dd099

File: source/includes/administration/_invoices.md (hunk 3 above)
Comment (tylerrwatsonn) [reply source/includes/administration/_invoices.md]: https://github.com/cloudops/cloudmc-api-docs/pull/418/commits/f116010e0dd55c35d39fda4a440e0d90944dd099

File: source/includes/administration/_invoices.md (hunk 4 above)
Comment (tylerrwatsonn) [reply source/includes/administration/_invoices.md]: https://github.com/cloudops/cloudmc-api-docs/pull/418/commits/f116010e0dd55c35d39fda4a440e0d90944dd099

**Review by jasondias9 — APPROVED** [review]

**Pull request merged**